src/map-app/app/presenter/map.ts

```diff
@@ -8,7 +8,7 @@ import { MapUI } from '../map-ui';
 
 export class MapPresenter extends BasePresenter {
   readonly view: MapView;
-  private previouslySelected: Initiative[] = [];
+  private currentlySelected: Initiative[] = [];
   
```

Comment:
Puzzling name change. Can't see a change in the logic. Was the name wrong for the logic?